src/ui/gtk/search_common.c

```diff
@@ -716,6 +716,10 @@ search_gui_clear_results(void)
 	search_gui_update_status(search);
 }
 
+
+static void
```

Comment:
Minor style issue: better spell-out forward declarations on a line by itself, like:

`static void search_gui_switch_search(struct search *search);`

and reserve the formatting you used to the actual definition of the routine.

Reply:
Will you take care o this? Should I "commit --amend" then "push -f" it? Open a new pull request?

Reply:
If you are not expert with git, I can take care of it after merging your commits in gtk-gnutella.
Or you can cleanup on your local branch by adding an extra commit.

If you are eager to play, you can rewrite yiour history with "git rebase -i HEAD^^^" and then move your last commit up and request a "fixup" merging with the original commit.  I have never done that on pull requests, so I have no idea what is going to happen with GitHub here if you fix your local history and force-push the changes to the pull request.

Let me know how you wish to proceed :-)

Reply:
I added one more commit fixing it.